tools/tsp-client/package.json

```diff
@@ -27,7 +27,7 @@
   "license": "MIT",
   "type": "module",
   "engines": {
-    "node": "^18.19.0 || >=20.6.0"
+    "node": ">=20.6.0"
```

Comment:
Yeah this is fine, since node18 was deprecated. Just run an npm install to update the package-lock.json file